.github/workflows/examples.yml

```diff
@@ -0,0 +1,26 @@
+name: Compile examples
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Configure CMake
+      working-directory: ${{github.workspace}}/examples
+      run: cmake -B ${{github.workspace}}/examples/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}}
+
+    - name: Build
+      working-directory: ${{github.workspace}}/examples
+      # Build your program with the given configuration
+      run: cmake --build ${{github.workspace}}/examples/build --config ${{env.BUILD_TYPE}}
```

Comment:
No need to specify the full path for the build folder. You're already inside examples.